src/support/scratch.c

```diff
@@ -400,16 +400,16 @@ __wt_scr_discard(WT_SESSION_IMPL *session)
 		if (*bufp == NULL)
 			continue;
 		if (F_ISSET(*bufp, WT_ITEM_INUSE))
-			__wt_errx(session,
-			    "scratch buffer allocated and never discarded"
+			__wt_errx(session, "%s",
```

Comment:
@dgottlieb This is the cause of the error I suspect. This is a new, 2nd call and the if statement needs {}.

Reply:
This was me, sorry @dgottlieb, fix pushed in c739f36.